draft-birkholz-rats-corim.md

```diff
@@ -789,10 +793,629 @@ Not included in the full CDDL definition are CDDL dependencies to CoSWID. The fo
+
+Initial registrations for the "CoMID Measurement Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.mkey | RFC-AAAA
+| 1 | comid.mvalue | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-comid-measurement-map-items title="CoMID Measurement Map Items Initial Registrations"}
+
+{: #iana-comid-measurement-values-map-items}
+## CoMID Measurement-Values-Map Items Registry
+
+This document defines a new registry titled "CoMID Measurement-Values Map".
+The registry uses integer values as index values for items in 'comid-measurement-values-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-measurement-values-map-items-reg-procedures title="CoMID Measurement-Values Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Measurement-Values Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.ver | RFC-AAAA
+| 1 | comid.svn | RFC-AAAA
+| 2 | comid.digests | RFC-AAAA
+| 3 | comid.flags | RFC-AAAA
+| 4 | comid.raw-value | RFC-AAAA
+| 5 | comid.raw-value-mask | RFC-AAAA
+| 6 | comid.mac-addr | RFC-AAAA
+| 7 | comid.ip-addr | RFC-AAAA
+| 8 | comid.serial-number | RFC-AAAA
+| 9 | comid.ueid | RFC-AAAA
+| 10 | comid.uuid | RFC-AAAA
+| 11-255 | Unassigned
+{: #tbl-iana-comid-measurement-values-map-items title="CoMID Measurement-Values Map Items Initial Registrations"}
+
+{: #iana-comid-verification-key-map-items}
+## CoMID Verification-Key-Map Items Registry
+
+This document defines a new registry titled "CoMID Verification-Key Map".
+The registry uses integer values as index values for items in 'comid-verification-key-map' CBOR maps.
+
+Future registrations for this registry are to be made based on {{RFC8126}} as follows:
+
+| Range             | Registration Procedures
+|---
+| 0-127    | Standards Action
+| 128-255  | Specification Required
+{: #tbl-iana-comid-verification-key-map-items-reg-procedures title="CoMID Verification-Key Map Items Registration Procedures"}
+
+All negative values are reserved for Private Use.
+
+Initial registrations for the "CoMID Verification-Key Map" registry are provided below.
+Assignments consist of an integer index value, the item name, and a reference to the defining specification.
+
+| Index | Item Name | Specification
+|---
+| 0 | comid.key | RFC-AAAA
+| 1 | comid.key-chain | RFC-AAAA
+| 2-255 | Unassigned
+{: #tbl-iana-comid-verification-key-map-items title="CoMID Verification-Key Map Items Initial Registrations"}
+
+{: #iana-comid-version-map-items}
```

Comment:
version-map isn't currently extensible. Extensions could break compatibility with other standards hence it should not be extensible.

Reply:
flagged for pruning